scripts/run-jest/index.js

```diff
@@ -0,0 +1,20 @@
+const spawn = require('child_process').spawn;
+const os = require('os');
+
+let regex = '';
+let jestcmd = '';
+switch (os.type()) {
+  case 'Linux':
+  case 'Darwin':
+    regex = `${process.env.npm_package_name}/tests`;
+    jestcmd = 'jest';
+    break;
+  case 'Windows_NT':
+    regex = `${process.env.npm_package_name}\\\\tests`;
+    jestcmd = 'jest.cmd';
+    break;
+  default:
```

Comment:
I am worried this check is too specific? What are the possible type returns? could we check for windows and if not windows use `jestcmd= 'jest' by default?

Reply:
Is the `.cmd` a by product of all bin scripts on windows machines? If so we might look into opening this up to accept a cmd paramater to toggle `jest || jest.cmd` or for example `wdio || wdio.cmd`

Reply:
Does this command correctly pass command line arguments ? For example, running from terra-button, does this run?

```bash
npm run tt:wdio -- -u
```